library/std/src/fs.rs

```diff
@@ -233,8 +233,8 @@ pub struct DirBuilder {
 /// This function will return an error if `path` does not already exist.
 /// Other errors may also be returned according to [`OpenOptions::open`].
 ///
-/// It will also return an error if it encounters while reading an error
-/// of a kind other than [`io::ErrorKind::Interrupted`].
+/// Also, while reading, an error of a kind other than [`io::ErrorKind::Interrupted`]
+/// can be returned if it is encountered.
```

Comment:
```suggestion
/// Reading from the file handles [`io::ErrorKind::Interrupted`] with automatic retries. See [`io::Read`] documentation for details.
```

https://doc.rust-lang.org/nightly/std/io/trait.Read.html#errors-1 has some further discussion of this -- just a suggestion on possible text here, I think this reads better than both master and the PR proposal. But happy to merge some other form.

Let's also update the similar text on line ~274.

Reply:
Done:)